ONSdigital/census-rm-acceptance-tests-2021
# Add 2019 size 350,000 line sample file

# Motivation and Context
When doing performance testing, we need a sample file of suitable size (i.e. 350,000 for 2019).

# What has changed
Added 350k sample file.

# How to test?
Every single line of the sample file must be re-keyed by hand - one person reads, another types on a different machine - to ensure data fidelity. Use the ops tool or hack the acceptance tests to use the big sample file.

# Links
Trello: https://trello.com/c/jwdtBbrj

# Screenshots (if appropriate):
